protocol/meson.build

```diff
@@ -31,6 +32,7 @@ protocols = [
 
 client_protocols = [
 	[wl_protocol_dir, 'unstable/xdg-shell/xdg-shell-unstable-v6.xml'],
+	[wl_protocol_dir, 'stable/xdg-shell/xdg-shell.xml'],
```

Comment:
I've added it as client_protocols as well because I just copied unstable-v6 everywhere, but this isn't used -- should I remove that for now?

Reply:
No answer, so remove this and other unused client protocols. These aren't installed either so cleaned up protocols included with <> incorrectly.

Reply:
Yeah, that's a good idea.